stix_shifter/src/json_to_stix/json_to_stix_translator.py

```diff
@@ -17,184 +17,152 @@ def convert_to_stix(datasource, map_data, data, transformers, options):
 
 class DataSourceObjToStixObj:
 
-    def __init__(self, datasource, dsToStixMap, transformers, options):
+    def __init__(self, datasource, ds_to_stix_map, transformers, options):
         self.datasource = datasource
-        self.dsToStixMap = dsToStixMap
+        self.dsToStixMap = ds_to_stix_map
         self.transformers = transformers
 
         # parse through options
         self.stix_validator = options.get('stix_validator', False)
 
-        self.common_props = observable.common_props
-        self.observation_props = observable.observation_props
+    @staticmethod
+    def _split_key(key, type_name=None):
+        try:
+            if type_name:
+                return key.split('.') if key.index('.') else None
+            else:
+                return key.replace('-', '_').split('.') if key.index('.') else None
+        except ValueError:
+            print('{0} cannot be split'.format(key))
```

Comment:
I'm not sure it's good practice to trigger an error here given that we expect this to happen for several of our input attributes (such as last_observed). I feel errors should be reserved for when things actually go wrong. It may be better to bring the logger back in so you can use the logging.debug call.

Reply:
You may also wish to keep the comment about why dashes are being replaced by underscores. You could just add this to the docstring assuming you go back to add some.